internal/api/api.go

```diff
@@ -271,6 +272,15 @@ func (a *API) GetServiceBinding(writer http.ResponseWriter, request *http.Reques
 		a.handleError(writer, err)
 		return
 	}
+	secrets, err := a.secretsForGivenServiceBindingID(sb.ID)
+	if err != nil {
+		a.handleError(writer, err)
+		return
+	}
+	if len(secrets.Items) > 0 {
```

Comment:
Why do we use only the first secret?

Reply:
A Service Binding should have only one secret with credentials, but the method to fetch the secret uses labels to get labeled secrets and there may be many copies of the secret in various namespaces. We just need one of them.